docs/sources/configure-server/about-server-api.md

```diff
@@ -213,6 +213,204 @@ curl -X POST \
 
 {{< /code >}}
 
+## Querying Profile Data
+
+There is one primary endpoint for querying profile data: `GET /pyroscope/render`.
+
+The search input is provided via query parameters.
+The output is typically a JSON object containing one or more time series and a flamegraph.
+
+### Query Parameters
+
+Here is an overview of the accepted query parameters:
+
+| Name           | Description                                                                           | Notes                                                |
+|:---------------|:--------------------------------------------------------------------------------------|:-----------------------------------------------------|
+| `query`        | contains the profile type and label selectors                                         | required                                             |
+| `from`         | UNIX time for the start of the search window                                          | required                                             |
+| `until`        | UNIX time for the end of the search window                                            | optional (default is `now`)                          |
+| `format`       | format of the profiling data                                                          | optional (default is `json`)                         |
+| `maxNodes`     | the maximum number of nodes the resulting flamegraph will contain                     | optional (default is `max_flamegraph_nodes_default`) |
+| `groupBy`      | one or more label names to group the time series by (doesn't apply to the flamegraph) | optional (default is no grouping)                    |
+
+#### `query`
+
+The `query` parameter is the only required search input. It carries the profile type and any labels we want to use to narrow down the output.
+The format for this parameter is similar to that of a PromQL query and can be defined as:
+
+`<profile_type>{<label_name>="<label_value>", <label_name>="<label_value>", ...}`
+
+Here is a specific example:
+
+`process_cpu:cpu:nanoseconds:cpu:nanoseconds{service_name="my_application_name"}`
+
+In a Kubernetes environment, a query could also look like:
+
+`process_cpu:cpu:nanoseconds:cpu:nanoseconds{namespace="dev", container="my_application_name"}`
+
+Note: Take a look at [profiling types]({{< relref "../ingest-and-analyze-profile-data/profiling-types" >}}) for more information.
```

Comment:
We're using an admonition shortcode for notes, cautions, tips, and warnings. For more information, see the [Shortcodes](https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition) documentation. 
```suggestion
{{% admonition type="note" %}}
Take a look at [profiling types]({{< relref "../ingest-and-analyze-profile-data/profiling-types" >}}) for more information.
{{% /admonition %}}
```